cmd/influx_tsm/b1/reader.go

```diff
@@ -96,7 +96,11 @@ func (r *Reader) Open() error {
 		}
 
 		measurement := tsdb.MeasurementFromSeriesKey(s)
-		for _, f := range r.fields[tsdb.MeasurementFromSeriesKey(s)].Fields {
+		fields := r.fields[tsdb.MeasurementFromSeriesKey(s)]
+		if fields == nil {
```

Comment:
Yeah, I was thinking about enhancing the stats, so at least a count of how many times we did this was displayed at the end.